javascript/javascript1/week1/Houseypricey.js

```diff
@@ -0,0 +1,45 @@
+//House price calculator
+const peterWidth = 8;
+const peterDeep = 10;
+const peterHeight = 10;
+const peterVolumeInMeters = peterWidth * peterDeep * peterHeight;
+const peterGardenSizeInMeters = 100;
+const peterHouseCost = 2500000;
+peterHousePrice = peterVolumeInMeters * 2.5 * 1000 + peterGardenSizeInMeters * 300;
+let peterHouse;
+if (peterHouseCost >= peterHousePrice)
```

Comment:
When you use `>=` operator then the `if` condition would be true in case of equality as well.
```suggestion
if (peterHouseCost > peterHousePrice)
```

Reply:
Yes, Made the changes